src/crypto/tls/handshake_messages.go

```diff
@@ -200,6 +201,17 @@ func (m *clientHelloMsg) marshal() []byte {
 					})
 				})
 			}
+			if m.delegatedCredentialSupported {
+				// Draft: https://datatracker.ietf.org/doc/draft-ietf-tls-subcerts/, Section 4.1
```

Comment:
Usually it is better to refer to a specific draft version since that will remain fixed over time, e.g. https://tools.ietf.org/html/draft-ietf-tls-subcerts-09#section-4.1

Reply:
Done ;)